README.md

```diff
@@ -17,7 +17,7 @@ English | [简体中文](README_zh-CN.md) | [日本語](README_ja_JP.md) | [ह
 
 ## Introduction
 
-Lagent is a lightweight open-source framework that allows users to efficiently build large language model(LLM)-based agents. It also provides some typical tools to augment LLM. The overview of our framework is shown below:
+Lagent is a lightweight, open-source framework that allows users to efficiently build large language model(LLM)-based agents. It also provides some typical tools to augment LLM. The overview of our framework is shown as below:
```

Comment:
It might be better to use `lightweight and open-source`, or `lightweight open-source`. `lightweight, open-source` seems does obey the English grammer.